workflows/Integration_of_InsightVM_and_Deep_Security_Virtual_Patching/workflow.spec.yaml

```
@@ -0,0 +1,33 @@
+extension: workflow
+products: ["insightconnect"]
```

Since both products are needed

```suggestion
products: ["insightconnect", "insightvm"]
```